ReactWindows/ReactNative/Modules/Toast/ToastModule.cs

```diff
@@ -0,0 +1,114 @@
+﻿using ReactNative.Bridge;
+using System.Collections.Generic;
+using Windows.Data.Xml.Dom;
+using Windows.UI.Notifications;
+
+namespace ReactNative.Modules.Toast
+{
+    using System.Diagnostics;
+    using System.IO;
+    using System.Threading.Tasks;
+    using Map = Dictionary<string, object>;
+    public partial class ToastModule : ReactContextNativeModuleBase
+    {
+        public ToastModule(ReactApplicationContext reactContext)
+            : base(reactContext)
+        {
+        }
+
+        private static readonly string DURATION_SHORT_KEY = "SHORT";
+        private static readonly string DURATION_LONG_KEY = "LONG";
+
+        public override string Name
+        {
+            get
+            {
+                return "ToastModule";
+            }
+        }
+
+        public Map getConstants()
+        {
+            return
+                    new Map
+                    {
+                        { DURATION_SHORT_KEY, DURATION_SHORT_KEY },
+                        { DURATION_LONG_KEY, DURATION_LONG_KEY },
+                    };
+        }
+
+        [ReactMethod]
+        public void show(string message, int duration)
+        {
+            /// TODO: UWP doesn't support a duration option.
+            ToastHelper.SendToast(message, "short");
+        }
+
+    }
+
+    enum durations : int { @short, @long }
+
+    internal class ToastHelper
+    {
+        internal static void SendToast(string message, string duration = "short")
+        {
+            var doc = MakeDoc("foobar", message).Result;
+            var notification = new ToastNotification(doc);
+
+            // Get the toast notification manager for the current app.
+            ToastNotificationManager.CreateToastNotifier().Show(notification);
+        }
+
+
+        static async Task<XmlDocument> MakeDoc(string title, string content, durations duration = durations.@short)
+        {
+            /// TODO: alt templates: https://msdn.microsoft.com/en-us/library/windows/apps/hh761494.aspx
+            using (var stream = new MemoryStream())
+            using (var reader = new StreamReader(stream))
+            {
+                await BuildNotificationXml(stream, title, content, duration);
+                stream.Position = 0;
+                var xml = await reader.ReadToEndAsync();
+                Debug.WriteLine("Toast XML");
+                Debug.WriteLine(xml);
+                var doc = new XmlDocument();
+                doc.LoadXml(xml);
+                return doc;
+            }
+
+        }
+
+        static async Task BuildNotificationXml(Stream stream, string title, string content, durations duration=durations.@short)
+        {
+            System.Xml.XmlWriterSettings settings = new System.Xml.XmlWriterSettings();
```

Comment:
[Windows.Data.Xml.Dom.XmlDocument\(https://msdn.microsoft.com/library/windows/apps/br206173) in use in UWP vs. System.Xml.Windows.Data.Xml.Dom